template/config/cypress-ct/package.json

```diff
@@ -1,7 +1,8 @@
 {
   "scripts": {
     "test:unit": "cypress run --component",
-    "test:unit:dev": "cypress open --component"
+    "test:unit:dev": "cypress open --component",
+    "test": "cypress run --component"
```

Comment:
maybe instead of repeating the command it could just be `"test": "npm run test:unit"`?
Same in every template

Reply:
OK! I thought at first that using `npm run` if the developer generated the project using `yarn` is a bad idea, but if not, I modify the templates.